### ISSUE TITLE
Fix and enable continuous run of the integration tests.

### ISSUE THREAD

**Issue by bashir2** [googlegenomics/gcp-variant-transforms#62]
There is currently a version of integration tests implemented (see [run_tests.py](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/testing/integration/run_tests.py)) and at some point in the past it has worked. But currently it does not run. We need to fix it, improve the tests and add some sort of validation (currently it does not validate the generated table). Finally we should implement some sort of continuous run for this integration test (it is not a small test).

**Comment by bashir2** [comment]
All the required changes to fix integration tests, add some minimal verifications to them, and run them automatically on every commit are done. Once the documentation changes of PR #97 is merged, this issue will be closed. The remaining nice to have but not critical enhancements are tracked in Issue #98.
